avocado/core/job.py

```diff
@@ -110,6 +110,24 @@ def resolutions_to_tasks(resolutions, config):
     return tasks
 
 
+def register_core_options():
+    """Register the few core options that the support the job operation."""
+    msg = (
+        "Sets the base log level of the output generated by the job, which "
+        "is also the base loging level for the --show command line option. "
+        "Any of the Python logging levels names are allowed here. Examples:"
+        " DEBUG, INFO, WARNING, ERROR, CRITICAL. For more information refer"
+        " to: https://docs.python.org/3/library/logging.html#levels"
+        )
+    settings.register_option(section='job.output',
+                             key='loglevel',
+                             default='DEBUG',
+                             help_msg=msg)
+
```

Comment:
I don't see much problem here. My only suggestion is the name of this method and the place where it is.

`register_core_options()` gives the impression that are all core options, but this is only related to job, right? `register_job_options()` IMO is a better name.

Reply:
`register_job_options()` works great for me too.